specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/examples/communityGalleryExamples/CommunityGallery_Get.json

```diff
@@ -13,6 +13,21 @@
         "type": "Microsoft.Compute/Locations/CommunityGallery",
         "identifier": {
           "uniqueId": "/CommunityGalleries/publicGalleryName"
+        },
+        "properties": {
+          "artifactTags": {
+            "ShareTag-CommunityGallery": "CommunityGallery"
+          },
+          "disclaimer": "https://aka.ms/community-gallery-disclaimer",
+          "communityMetadata": {
+            "publisherUri": "http://microsoft.com",
+            "publisherContact": "sig@microsoft.com",
+            "eula": "http://microsoft.com",
```

Comment:
Consider that if these show up in examples, people may copy them, incorrectly or accidentally, or just out of laziness. It might be better if they copied something obviously fake than microsoft.com, if so?

Reply:
A separate PR here, https://github.com/Azure/azure-rest-api-specs/pull/25318/files#diff-7116527d8a6efb6fe945b7f461b4d0f7404602953b520d5d937a87e4221364e1 , addressed all of your comments.